Codeinwp/otter-blocks
# Initial PHPStan Version

Closes #.


### Summary


This PR adds PHPStan to Otter.

### Screenshots 

----

### Test instructions

We just have to make sure the Otter plugins works normally and this build doesn't bring any issues that don't already exist in the master branch. We can do a quick test of Otter to make sure things are fine in addition we can test the below list in particular to make sure it works fine (if there are some issues, they're not new but already exist):

- Make sure Block-generated CSS & Fonts are loaded properly in frontend
- Make sure Animations work properly
- Form Block Subscription & Sending works properly
- Query String Dynamic Content
- Posts Grid Categories & Featured Post
- Sharing Icons
- Location Based & Woo Total Cart Value based Block Conditions
- ACF field in Posts Block

---- 

### Checklist before the final review
